src/request/create/index.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="Content-Security-Policy"
+          content="default-src 'self' 'unsafe-eval' https://cdn.nimiq.com;
+          img-src http: https: blob: data:; child-src 'self' blob:; style-src 'self';">
+
+    <script defer src="https://cdn.nimiq.com/web-offline.js"></script>
+    <script defer src="../../errors.js"></script>
+    <script defer src="../../lib/EncryptionType.js"></script>
+    <script defer src="../../lib/TransactionType.js"></script>
+    <script defer src="../../lib/Key.js"></script>
+    <script defer src="../../lib/KeyStoreIndexeddb.js"></script>
+    <script defer src="../../lib/RpcServer.js"></script>
+    <script defer src="../../lib/AddressUtils.js"></script>
+    <script defer src="../../lib/Utf8Tools.js"></script>
+    <script defer src="../../lib/Iqons.js"></script>
+    <script defer src="../../lib/BrowserDetection.js"></script>
+    <script defer src="../../lib/I18n.js"></script>
+    <script defer src="../../translations/index.js"></script>
+    <script defer src="../../lib/MnemonicPhrase.js"></script>
+    <script defer src="../../common.js"></script>
+    <script defer src="../../components/PinInput.js"></script>
+    <script defer src="../../components/PassphraseInput.js"></script>
+    <script defer src="../../components/Identicon.js"></script>
+    <script defer src="../../components/RecoveryWords.js"></script>
+    <script defer src="../../components/ValidateWords.js"></script>
+    <script defer src="../PopupApi.js"></script>
+    <script defer src="./ChooseIdenticon.js"></script>
+    <script defer src="CreateHigh.js"></script>
+    <script defer src="CreateLow.js"></script>
+    <script defer src="CreateApi.js"></script>
+    <script defer src="index.js"></script>
+
+    <link href="../../nimiq-style.css" rel="stylesheet">
+    <link rel="stylesheet" href="../../common.css">
+    <link rel="stylesheet" href="../../components/PinInput.css">
+    <link rel="stylesheet" href="../../components/PassphraseInput.css">
+    <link rel="stylesheet" href="../../components/RecoveryWords.css">
+    <link rel="stylesheet" href="../../components/ValidateWords.css">
+    <link rel="stylesheet" href="./privacy-agent.css">
```

Comment:
The PrivacyAgent is needed in other requests, too. It should be made a "component".

Reply:
To be done in svub's import request PR.